src/DotLiquid/Tags/If.cs

```diff
@@ -20,7 +20,8 @@ namespace DotLiquid.Tags
     public class If : DotLiquid.Block
     {
         private string SyntaxHelp = Liquid.ResourceManager.GetString("IfTagSyntaxException");
-        private static readonly Regex Syntax = R.B(R.Q(@"({0})\s*([=!<>a-zKV_]+)?\s*({0})?"), Liquid.QuotedFragment);
+        private string TooMuchConditionsHelp = Liquid.ResourceManager.GetString("IfTagTooMuchConditionsException");

```

Comment:
The TooMuchConditionsHelp string is from the master. I could remove it if desired, but that would conflict with the master repo: https://github.com/dotliquid/dotliquid/blob/master/src/DotLiquid/Tags/If.cs#L23